### PR TITLE
Add Makefile and agencies.json build

### PR DESCRIPTION
**Pull request by pkarman**
Makefile should ease common tasks

agencies.json build is the prototype model

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+## About
+
+DOJ Office of Information Policy has posted a [blog post about the project](https://www.justice.gov/oip/blog/oip-seeks-your-participation-development-national-foia-portal).
+
+## Build the prototypes
+
+```bash
+ % make setup
+ % make build
+ % make run
```
Comment (adborden) [inline README.md:12]: I don't see this step in the Makefile, is it missing?

File: prototypes/form-wizard/_plugins/jekyll-agency-model-builder.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+require 'pp'
+require 'json'
+
+module Jekyll
+  class AgencyJsonGenerator < Generator
+    safe true
+
+    DEFAULT_YAML_DIR = __dir__ + '/../../../../2015-foia/contacts/data'
```
Comment (adborden) [inline prototypes/form-wizard/_plugins/jekyll-agency-model-builder.rb:8]: :+1: maybe we want to mention this in the README, that you want to clone the directory next to `foia-recommendations`, right?

Or that you can override it via env var.

File: prototypes/form-wizard/_plugins/jekyll-agency-model-builder.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    OUTPUT_DIR = 'assets'
+    OUTPUT_FILE = '_site/' + OUTPUT_DIR + '/agencies.json'
+
+    def generate(site)
+      agencies = load_yaml_files(ENV.fetch('FOIA_AGENCY_DIR', DEFAULT_YAML_DIR))
```
Comment (adborden) [inline prototypes/form-wizard/_plugins/jekyll-agency-model-builder.rb:13]: Nice.

File: README.md (hunk 1 above)
Comment (pkarman) [reply README.md:12]: good catch. added.

File: prototypes/form-wizard/_plugins/jekyll-agency-model-builder.rb (hunk 2 above)
Comment (pkarman) [reply prototypes/form-wizard/_plugins/jekyll-agency-model-builder.rb:8]: done.

**Review by adborden — APPROVED** [review]

**Pull request merged**